master/internal/resourcemanagers/kubernetes_resource_manager.go

```diff
@@ -199,8 +205,39 @@ func (k *kubernetesResourceManager) receiveRequestMsg(ctx *actor.Context) error
 	case sproto.SetGroupMaxSlots:
 		k.getOrCreateGroup(ctx, msg.Handler).maxSlots = msg.MaxSlots
 
-	case sproto.SetGroupWeight, sproto.SetGroupPriority:
-		// SetGroupWeight and SetGroupPriority are not supported by the Kubernetes RP.
+	case sproto.SetGroupWeight:
+		// SetGroupWeight is not supported by the Kubernetes RP.
```

Comment:
is this temporary? if not we should return an error

Reply:
returning an error causes further errors upstream - this would require a lot more changes and special handling for k8s cases.